assets/css/stripeStyle.css

```diff
@@ -573,3 +579,10 @@ table tr {
 .coupon-message {
 	height: 8px;
 }
+.coupon-notice {
+	color: #ffd600;
+}
+
+.description {
+	font: 400 14px/19px "Open Sans", sans-serif;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218)